compiler/p/env/OMRCPU.cpp

```diff
@@ -45,23 +152,109 @@ OMR::Power::CPU::isTargetWithinIFormBranchRange(intptr_t targetAddress, intptr_t
           range >= self()->maxIFormBranchBackwardOffset();
    }
 
-bool 
-OMR::Power::CPU::getSupportsHardwareSQRT()
+bool
+OMR::Power::CPU::supportsFeature(uint32_t feature)
```

Comment:
Is this overlapping somehow with the various separate feature testing such as SQRT/HTM/...?

Reply:
Yes, I think the end goal is to use this instead of the hardcoded queries. I originally wanted to do that in a separate PR but since you brought it up, I could just replace them in this PR.